src/app/feed/feed-card/feed-card.component.ts

```diff
@@ -131,31 +135,32 @@ export class FeedCardComponent implements OnInit {
 		}
 	}
 
-	private sanitizeAndEmbedURLs(links) {
-		links.forEach((link, i) => {
-			const videoid = links[i].match(/(?:https?:\/{2})?(?:w{3}\.)?youtu(?:be)?\.(?:com|be)(?:\/watch\?v=|\/)([^\s&]+)/);
+	private sanitizeVideoURLs() {
+		this.feedItem.videos.forEach((video) => {
+			const videoid = video.match(/(?:https?:\/{2})?(?:w{3}\.)?youtu(?:be)?\.(?:com|be)(?:\/watch\?v=|\/)([^\s&]+)/);
```

Comment:
@hemantjadon , media is also coming from twitter embedded videos. try queries like http://loklak.org/api/search.json?q=video&filter=video&count=100&source=twitter